### PR TITLE
Additional information screens

### PR DESCRIPTION
**Pull request by HUSKUT**
this shouldve been 2 branches in the beginning but i forgor :skull:

:shipit:

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -41,6 +45,7 @@ export default function App() {
         <Stack.Navigator
           initialRouteName={isLoggedIn ? 'HomeScreen' : 'WelcomeScreen'}
         >
+          {/*<Stack.Navigator initialRouteName={'SelectSignUpTypeScreen'}>*/}
```
Comment (SophiaH67) [inline App.tsx]: Commented out line

File: components/SetupWrapper.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export default function SetupWrapper({ children }: SetupWrapperProps) {
             position: 'absolute',
           }}
         >
-          {ApiUrl}
+          {/*{ApiUrl}*/}
```
Comment (SophiaH67) [inline components/SetupWrapper.tsx]: Why this? In testing I thought it was pretty useful seeing the api url used

File: screens/SelectSignUpTypeScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          style={styles.blueButton}
+          onPress={() => {
+            //@ts-expect-error
+            navigate('SignUpScreen', {
+              Type: 'Company',
+            });
+          }}
+        >
+          <SvgUri
+            uri={'https://icons.getbootstrap.com/assets/icons/building.svg'}
```
Comment (SophiaH67) [inline screens/SelectSignUpTypeScreen.tsx:57]: Should we maybe download this asset and import it into the project, or was that too much of a PITA?

**Review by SophiaH67 — CHANGES_REQUESTED** [review]
Some small changes, rest looks good (including the use of any)

File: screens/SelectSignUpTypeScreen.tsx (hunk 3 above)
Comment (HUSKUT) [reply screens/SelectSignUpTypeScreen.tsx:57]: way too much of a pain in the ass for current project imo, although it goes against my goal its just hassle for something simple

File: components/SetupWrapper.tsx (hunk 2 above)
Comment (HUSKUT) [reply components/SetupWrapper.tsx]: Fairs, I guess I got annoyed with the text and/or removed it for demo purposes and totally forgot it

File: App.tsx (hunk 1 above)
Comment (HUSKUT) [reply App.tsx]: Looked right over that one, Thanks for pointing it out!

**Review by SophiaH67 — APPROVED** [review]
:skull:

**Comment by HUSKUT** [comment]
:shipit:

**Pull request merged**